### PR TITLE
Cleans /src/utils/charts. #1018

### PR DESCRIPTION
**Pull request by knod**

### CODE REVIEW

File: src/utils/charts/chartFormatting.js [hunk 1]
```diff
@@ -1,23 +1,22 @@
 import _ from 'lodash';
 
-/** Helper functions to format vlaues
+/** Helper functions to format vlaues for charts
```
Comment (turnerhayes) [inline src/utils/charts/chartFormatting.js]: "to format vlaues" => "to format *values*"

**Review by turnerhayes — CHANGES_REQUESTED** [review]
One minor typo (preexisting, but might as well fix it while we're cleaning stuff up here).

File: src/utils/charts/chartFormatting.js (hunk 1 above)
Comment (knod) [reply src/utils/charts/chartFormatting.js]: Done! Better?

**Review by turnerhayes — APPROVED** [review]

File: src/utils/charts/chartFormatting.js (hunk 1 above)
Comment (turnerhayes) [reply src/utils/charts/chartFormatting.js]: :+1:

**Pull request merged**